deploy/registration-service/registration-service.yaml

```diff
@@ -32,6 +32,7 @@ objects:
           - toolchain.dev.openshift.com
         resources:
           - masteruserrecords
+          - usertiers
```

Comment:
You don't need to get tiers from reg-service with the new approach, do you?

Reply:
Fixed